app/api_1_0/utils.py

```diff
@@ -1,24 +1,39 @@
 whitelists = {
-	'api.revisions': ['comment','anon','pageid','tags','timestamp','userid','revid','contentformat','contentmodel','extraction_date','parentid','title','_id','size','user','minor']
+	'api.revisions': ['comment','anon','pageid','tags','timestamp','userid','revid','contentformat','contentmodel','extraction_date','parentid','title','_id','size','user','minor'],
+	'filter_attributes':['title','pageid','user','userid','tag','size','sizematch','date','datestart','dateend'],
+	'mode_attributes':['title','user','size','userid','date']
 }
 
 param_type = {
 	'pageid': 'int',
 	'userid': 'int',
 	'revid': 'int',
 	'size': 'int',
+	'sizematch':'int',
 	'userid': 'int'
 }
 
-def filter_params(request):
+def filter_params(request,whitelist_category=None):
```

Comment:
No need to add `whitelist_category`, use the `request` param and add the proper endpoint entries to `whitelists ` variable